README.md

```diff
@@ -10,77 +10,158 @@ be stable.
 
 ## Code Samples
 
-|Code Sample    |Supported Intel(r)   Architecture(s)    	|Description 	|
+|Code Sample    |Supported Intel®   Architecture(s)    	|Description 	|
```

Comment:
I think the generally accepted way to get a (R) symbol in Markdown is with &reg

Reply:
fixed it